src/Main.java

```diff
@@ -1,5 +1,70 @@
 public class Main {
+    private static final Employee[] empl = new Employee[10];
```

Comment:
```suggestion
    private static final Employee[] EMPL = new Employee[10];
```
поля static И final надо называть в верхнем регистре